pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
@gfyoung See also my comment of yesterday (https://github.com/pydata/pandas/pull/13425#issuecomment-229207939)

> Your latest change is less convenient but maybe more flexible than it was before (now you have to specify the exact dtype you want, instead of just saying "give me the smallest possible int dtype")